AdguardTeam/AdguardFilters
# browse.gmarket.co.kr - Tracking

### URL
https://browse.gmarket.co.kr/m/search?keyword=7days

### How to reproduce
1. Visit the provided URL with responsive design mode of a browser.
2. Scroll the website down for a while.
3. If the following button appears, please click the button.
    <details>
    <summary>Screenshot</summary>
    
     ![Screensho](https://user-images.githubusercontent.com/98787049/162630668-d28a489f-3454-42d8-b6fe-cd30e562a8a7.png)

      </details>

4. Finally, check network section of devtool.

### Code preview
<details>
<summary>Screenshot</summary>

![Screenshot](https://user-images.githubusercontent.com/98787049/162630547-5c3816ca-bf1e-4045-bc08-7eeb3373532e.png)

</details>

### Similarweb Web Traffic Report
https://www.similarweb.com/website/gmarket.co.kr/#traffic

### Related Issue
https://github.com/AdguardTeam/AdguardFilters/issues/78959
https://github.com/List-KR/List-KR/issues/285
